ui/pages/unlock-page/unlock-page.component.js

```diff
@@ -93,7 +93,7 @@ export default class UnlockPage extends Component {
     } catch ({ message }) {
       this.failed_attempts += 1;
 
-      if (message === 'Incorrect password') {
+      if (message.startsWith('Incorrect password')) {
```

Comment:
Hmm, it's not clear what could have necessitated this change

Reply:
Leftover from WIP attempts, I think. Will see if these can be reverted to keep the changes relevant.